packages/web-components/fast-foundation/src/tabs/tabs.template.ts

```diff
@@ -15,6 +15,7 @@ export const tabsTemplate: FoundationElementTemplate<ViewTemplate<Tabs>, TabsOpt
     <template class="${x => x.orientation}">
         ${startSlotTemplate(context, definition)}
         <div class="tablist" part="tablist" role="tablist">
+            <slot name="tabs-start"></slot>
```

Comment:
I think an issue here is that the play/pause button doesn't have a role of `tab`, does it? We'll likely run into issues with something which is not a role of `tab` being contained in a `tablist`.

Reply:
Note: I think this is why start/end wrap the `tablist` rather than sit within it.